Code_Patch1.gs

```diff
@@ -0,0 +1,25 @@
+function onOpen() {
+  SpreadsheetApp.getUi()
+  // Create a menu called Ali with a button called run script
+      .createMenu('Ali')
+      .addItem('Run Script', 'alertWithMessage')
+      .addToUi();
+}
+
+function alertWithMessage(message) { 

```

Comment:
Message in this case means the message that you want to shop users. So change the way the input variable `message` is used. So add this to your alert and remove the hardcoded message 'Click button yes or no'. You should create another function that calls your alertWithMessage function.

As a rule of thumb input variables are not modified when they are passed in. You make modifications with internal variable. There are exceptions but not in the case of this ticket.